app/models/manager_refresh/inventory_collection.rb

```diff
@@ -9,7 +9,7 @@ class InventoryCollection
 
     delegate :each, :size, :to => :to_a
 
-    def initialize(model_class, manager_ref: nil, association: nil, parent: nil, strategy: nil, saved: nil,
+    def initialize(model_class: nil, manager_ref: nil, association: nil, parent: nil, strategy: nil, saved: nil,
                    custom_save_block: nil, delete_method: nil, data_index: nil, data: nil, dependency_attributes: nil,
                    attributes_blacklist: nil, attributes_whitelist: nil, complete: nil, update_only: nil,
                    check_changed: nil, custom_manager_uuid: nil, custom_db_finder: nil, arel: nil)
```

Comment:
yeah, I think will rater need to create a post validation, cause there are probably groups that should go together. I'll do that after the Docs. :-)